src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig

```diff
@@ -94,5 +94,13 @@
         </button>
       </div>
     </div>
+  {% elseif merchandiseReturnEnabled %}
+    <div class="form-inline d-inline-block form-horizontal ml-2">
```

Comment:
This button should be integrated in the block a bit above `{% if merchandiseReturnEnabled %}` along with all the buttons related to the merchandise return feature, see https://github.com/PrestaShop/PrestaShop/pull/17416/files#diff-9ec92f94e56724fb3be768b86bcd689eR76

Reply:
Ok, to be fixed once your standard refund PR is merged.

Reply:
done